validator/client/src/test/java/tech/pegasys/teku/validator/client/signer/SignerTest.java

```diff
@@ -58,7 +58,7 @@ public void shouldSignBlock1() {
     final BeaconBlock block = dataStructureUtil.randomBeaconBlock(10);
     final BLSSignature signature = dataStructureUtil.randomSignature();
     final Bytes expectedSigningRoot =
-        Bytes.fromHexString("0xfa8b3cfed0268ed15e354e84db5558eb76ad30737a86d6d057615e331ff30d44");
+        Bytes.fromHexString("0xf6c68e87f3dbbe3d05de8eae204396ddea7fe13ffad0008f06748c9aa23fcc05");
```

Comment:
These signing roots changed because the epoch included in the `AttestationData` generated (which is included in the block body) is different (it's now within the next 12,000 years).